Source/UIViewExtension.swift

```diff
@@ -0,0 +1,54 @@
+/*   Copyright 2019-2020 Prebid.org, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import Foundation
+
+extension UIView {
+    
+    func an_isAtLeastHalfViewable() -> Bool{
```

Comment:
what does prefix `an_` mean ?

Reply:
an_ for Appnexus but updated the same with pb_ which is for Prebid